chainguard-dev/edu
# Documentation for CVE Visualizations feature.

*DO NOT MERGE BEFORE CVE VIZ IS IN BETA*

## Type of change
Add docs for CVE Visualizations

### How should this PR be tested?
Please check the language. Feel free to click through the feature as well, you may need to enable it first by using this URL:
https://console.chainguard.dev/reports?feature.compare=true

[Direct link to preview](https://deploy-preview-1940--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/working-with-images/cve_visualizations/)
